resources/css/site.css

```diff
@@ -603,3 +603,49 @@ a.text-center.btn.btn-outline-light.btn__book {
 .card-img-top {
     height: 100% !important;
 }
+
+.p-heading {
+    font-weight: 500;
+}
+
+.p-container {
+    flex-direction: column-reverse;
+    padding: 40px;
+}
+
+.p-container--single {
+    color:#299942;
+}
+
+.p-container--joint {
+    color: #165181;
+    margin-bottom: 16px;
+}
+
+.p-image {
+    max-width: 30%;
+    margin: 0 auto;
+}
+
+.p-image--single {
+    filter: invert(47%) sepia(44%) saturate(738%) hue-rotate(82deg) brightness(93%) contrast(89%);
```

Comment:
Can you please add a comment to explain what the filter does? I know it changes the colour of the icon, but I only know because we talked about it :) 

If you have a relevant article about how this works, then please feel free to include the URL for it in the CSS comment.

Reply:
Filters are removed now.